### PR TITLE
Kubecon NA blog post

### PR DESCRIPTION
**Pull request by deepthi**
Signed-off-by: deepthi <deepthi@planetscale.com>

### CODE REVIEW

File: content/en/blog/2020-12-02-vitess-at-kubecon-na.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+This was our big day. Not only did we have both Vitess talks on this day, but we also had mentions in keynotes.
+[Link to Keynotes]
+[Link to Slack talk]
+[Link to maintainer talk]
+
+Our project office hours featuring Sugu and Rafael drew 120 attendees.
+
+## Day 3
+Another quiet day. We had our monthly public meeting on this day, so we did not host office hours.
+
```
Comment (askdba) [inline content/en/blog/2020-12-02-vitess-at-kubecon-na.md]: Here's the link for the Monthly Call Recording https://zoom.us/rec/share/A8tOSvILYkUx65bizFvkph7_aFNOyWkEr3VxqqJl5XealI7aKSodVVdaJ5I-mArd.ftVtXObha-BCERxo Access Passcode: AKi00k&@

File: content/en/blog/2020-12-02-vitess-at-kubecon-na.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## Day 4
+We hosted our last project office hours hosted by Alkin and featuring Deepthi and Paul Hemberger. 65 people attended the session.
+
+## Project Booth
+We had over 350 unique visitors to the project booth. We had maintainers available to answer questions but most visitors chose to view documents and not engage with booth staff.
```
Comment (askdba) [inline content/en/blog/2020-12-02-vitess-at-kubecon-na.md]: Maybe we can include infographics published by CNCF in this part to make it visually appealing.

File: content/en/blog/2020-12-02-vitess-at-kubecon-na.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+author: 'Deepthi Sigireddi'
+date: 2020-12-02
+slug: '2020-12-02-vitess-at-kubecon-na'
+tags: ['conference']
```
Comment (askdba) [inline content/en/blog/2020-12-02-vitess-at-kubecon-na.md]: Please always add MySQL to the tags for our Blog RSS feed for planet.mysql.com and planet for community sites.

File: content/en/blog/2020-12-02-vitess-at-kubecon-na.md (hunk 2 above)
Comment (deepthi) [reply content/en/blog/2020-12-02-vitess-at-kubecon-na.md]: I didn't want to steal that content so I didn't do this part :)

**Comment by deepthi** [comment]
@askdba should we embed the video for the vitess talks instead of links?

**Comment by askdba** [comment]
@deepthi just a few minor edits on the intro below.  Can we add Twitter handles as hyperlinks under names? This way if people want to reach out they can. If Twitter handles not exist we can use GitHub handle instead. Otherwise, this looks good to me as a recap. 


"The Vitess team had a successful presence at Kubecon + CloudNativeCon North America 2020. This year’s event was in a virtual format. It ran between November 17th and 20th. We had two talks, a project booth, and three office hours sessions. In addition, we had mentioned two major keynote talks."

**Review by shlomi-noach — APPROVED** [review]
:steam_locomotive:

**Comment by deepthi** [comment]
> @deepthi just a few minor edits on the intro below. Can we add Twitter handles as hyperlinks under names? This way if people want to reach out they can. If Twitter handles not exist we can use GitHub handle instead. Otherwise, this looks good to me as a recap.
> 
> "The Vitess team had a successful presence at Kubecon + CloudNativeCon North America 2020. This year’s event was in a virtual format. It ran between November 17th and 20th. We had two talks, a project booth, and three office hours sessions. In addition, we had mentioned two major keynote talks."

Can you take another look? I added twitter/github links.

**Pull request merged**